frontend/public/actions/ui.ts

```diff
@@ -282,6 +284,19 @@ export const sortList = (
 
   return action(ActionType.SortList, { listId, field, func, orderBy });
 };
+export const setTableColumns = (id: string, selectedColumns: Set<string>) => {
+  let currentColumns = {};
+  try {
+    currentColumns = JSON.parse(localStorage.getItem(COLUMN_MANAGEMENT_LOCAL_STORAGE_KEY)) || {};
+  } catch (e) {
+    // Error parsing the data, do not store the current filters
```

Comment:
Comment is no longer accurate

Reply:
@spadgett yes this is working correctly...thanks for improving this.

Reply:
updated comment